cmd/report.go

```diff
@@ -253,13 +252,16 @@ func init() {
 		"verify the upgrade compatibility of every cookbook",
 	)
 	// adds the cookbooks command as a sub-command of the report command
-	// => chef-analyze report cookbooks
 	reportCmd.AddCommand(reportCookbooksCmd)
 
 	// adds the nodes command as a sub-command of the report command
-	// => chef-analyze report nodes
 	reportCmd.AddCommand(reportNodesCmd)
 
+	// adds the upload command as a hidden sub-command of the report command
+	reportCmd.AddCommand(uploadCmd)
```

Comment:
This is better, and should let me get rid of the weirdness I had for forcing the usage text in the cmd defs.